homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: #221F1F;
+}
+
+.info {
+  max-width: 575px;
+}
+
+.description {
+  font-size: 48px;
+  font-weight: 700;
+}
+
+.text {
+  font-size: 22px;
```

Comment:
please add a line-heigh from the desing
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/7c6956a4-c843-4e07-a7eb-667770c1574d)

Reply:
fix: add line-height for description and text